src/app/containers/Ad/Amp/index.jsx

```diff
@@ -91,40 +113,55 @@ export const AMP_ACCESS_FETCH = service => {
 };
 
 // eslint-disable-next-line react/prop-types
-const AmpAd = ({ service }) => {
+const AmpAd = () => {
+  const { ads, dir, script, service, footer } = useContext(ServiceContext);
+  const label = pathOr('Advertisement', ['advertisementLabel'], ads);
+  const links = pathOr(null, ['links'], footer);
+  const labelLink = links && pathOr(null, ['href'], links[3]);
```

Comment:
I'm not sure about accessing the third element of the footer links array here, there's nothing to ensure that this is going to the the cookie link. Could we either add a new item to config, or just use the string here for now if the current requirement is to always use the same link, and then add this to config if this needs to become configurable?

Reply:
Since we're using the same link for all services, using a string would be a much simpler approach.